OutSystems/outsystems-ui
# ROU-4629: Fix an issue on datepicker to match values on InitialDate

This PR is for fix an issue on datepicker to match values the platform input and Datepicker InitialDate

### What was happening
- The date that are being applied on platform input, doesn't match with datepicker's date and when the date format is different between server and datepicker.

### What was done
- Created a new method to check the InitialDate that needs to be applied on DatePicker
- Validate in platform input contains value and set as a InitialDate, if the DatePicker doens't have a date defined

### Test Steps

1. Open test page
2. Check if the date on input is the same of the platfomr hidden input
3. Open datepicker and select a new date
4. Check again the dates

### Checklist
